### ISSUE TITLE
decommissioning python2 binding

### ISSUE THREAD

**Issue by brancomat** [ARPA-SIMC/dballe#127]
this depends on residual porting to python3:

- [x] rmap (@pat1)
- [ ] internal operating procedures (@brancomat @edigiacomo)
- [x] oceanographic procedures

any other reason to postpone python2 decomissioning for dballe should be added in this issue

**Comment by lidiabressan** [comment]
I have some scripts not yet converted to python3.

**Comment by edigiacomo** [comment]
I talked to @lidiabressan: the oceanographic procedures aren't a blocking issue. We can update the servers and @lidiabressan can work on her workstation with a freezed version of dballe until she has completed the porting to python3.

**Comment by spanezz** [comment]
The branch `issue122` already has no python2 bindings. This can be closed once `issue122` is merged into master.

**Comment by spanezz** [comment]
This can be closed now.
